src/components/chat/ChatElement.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+import { defineProps, computed } from "vue";
+import { ConversationMessage } from "src/MooLite/plugins/WhisperManager/ConversationMessage";
+
+const props = defineProps<{
+    message: ConversationMessage;
+}>();
+</script>
+
+<template>
+    <div
+        class="flex flex-row items-end space-x-1 w-full"
+        :class="{
+            'justify-end': !message.isInbound,
+        }"
+    >
+        <svg
```

Comment:
Please refactor svgs to custom components so they are easier to understand semantically

Reply:
And we can reuse them later :)